source/worksheet/page_setup.cpp

```diff
@@ -105,16 +105,19 @@ void page_setup::fit_to_width(bool fit_to_width)
     fit_to_width_ = fit_to_width;
 }
 
-void page_setup::scale(double scale)
+void page_setup::scale(unsigned int scale)
 {
-    scale_ = scale;
+    if (scale >= 10 && scale <= 400)
```

Comment:
Maybe it is better to clip the value to the [10, 400] range, instead of just ignoring those (invalid) requests? By clipping, we honor the user request as best as we could.